src/api/client.py

```
@@ -2,6 +2,9 @@
 import json
 import asyncio
 import weakref
+import traceback
+import asyncio
```

Ruff (0.8.2)

Redefinition of unused `asyncio` from line 3

Remove definition: `asyncio`

(F811)